### PR TITLE
Implement project-log

### PR DESCRIPTION
**Pull request by fsouza**
It just parses the flags and sets the proper parameters for the tsuru
app-log command

### CODE REVIEW

File: log_test.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+		t.Fatal(err)
+	}
+	var stdout, stderr bytes.Buffer
+	ctx := cmd.Context{Stdout: &stdout, Stderr: &stderr}
+	cli := cmd.NewClient(http.DefaultClient, &ctx, &cmd.Manager{})
+	err = c.Run(&ctx, cli)
+	if err == nil {
+		t.Fatal("got unexpected <nil> error")
+	}
+}
```
Comment (aflag) [inline log_test.go:100]: Is it difficult to test the case where the log does succeed?

**Comment by aflag** [comment]
I have one comment. But I already have already built and tested it. So, if it is not possible or if it is very hard to do a test in which log succeeds, I'm ok with this.

File: log_test.go (hunk 1 above)
Comment (fsouza) [reply log_test.go:100]: @aflag I'll add a test for the success case! Thanks for reviewing!

**Comment by ehault** [comment]
👍

**Comment by fsouza** [comment]
@aflag I just added the test, please take another look!

**Comment by aflag** [comment]
:+1:

**Pull request merged**